frontend/src/lib/components/ActivationSidebar/activationLogic.ts

```diff
@@ -0,0 +1,266 @@
+import { kea, path, actions, selectors, connect, reducers, listeners, events } from 'kea'
+import { loaders } from 'kea-loaders'
+import { router, urlToAction } from 'kea-router'
+import api from 'lib/api'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
+import { inviteLogic } from 'scenes/organization/Settings/inviteLogic'
+import { membersLogic } from 'scenes/organization/Settings/membersLogic'
+import { pluginsLogic } from 'scenes/plugins/pluginsLogic'
+import { teamLogic } from 'scenes/teamLogic'
+import { navigationLogic } from '~/layout/navigation/navigationLogic'
+import { EventDefinitionType, TeamBasicType } from '~/types'
+import type { activationLogicType } from './activationLogicType'
+import { urls } from 'scenes/urls'
+
+export enum ActivationTasks {
```

Comment:
We should def add tasks for creating a dashboard, viewing an insight, etc. Most people are signing up for product analytics and we want to make sure they are actually getting a good tour of that product before we push them onto something else like recordings, esp since our activation metric depends on that.

Reply:
Done done done